airbyte-integrations/connectors/source-sonar-cloud/source_sonar_cloud/sonar_cloud.yaml

```
@@ -0,0 +1,102 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  components_selector:
+    extractor:
+      field_pointer: ["components"]
+  issues_selector:
+    extractor:
+      field_pointer: ["issues"]
+  metrics_selector:
+    extractor:
+      field_pointer: ["metrics"]
+  requester:
+    url_base: "https://sonarcloud.io/api"
+    http_method: "GET"
+    request_options_provider:
+      request_parameters:
+        organization: "{{ config['organization'] }}"
+        createdAfter: "{{ config['start_date'] }}"
+        createdBefore: "{{ config['end_date'] }}"
```

```suggestion
        createdBefore: "{{ config['end_date'] or now_utc() }}"
```
Check macros here: https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/declarative/interpolation/macros.py

---

That will not work for this case, the api waiting date in specific format: YYYY-MM-DD, the macro return date-time format:
2022-10-26+21%3A02%3A17.320564%2B00%3A00